items.json

```diff
@@ -5995,6 +5995,42 @@
       }
     ]
   },
+  {
+    "id": "crpg_bolt_f",
+    "name": "Heavy Piercing Bolts",
+    "culture": "Empire",
+    "type": "Bolts",
+    "price": 7815,
+    "weight": 0.08,
+    "tier": 9.736687,
+    "requirement": 0,
+    "flags": [
+      "ForceAttachOffHandPrimaryItemBone"
+    ],
+    "weapons": [
+      {
+        "class": "Bolt",
+        "itemUsage": "",
+        "accuracy": 100,
+        "missileSpeed": 10,
+        "stackAmount": 5,
+        "length": 37,
+        "balance": 0.0,
+        "handling": 0,
+        "bodyArmor": 0,
+        "flags": [
+          "AttachAmmoToVisual",
+          "Consumable"
+        ],
+        "thrustDamage": 14,

```

Comment:
lower all light crossbows damage by 1 for this. crossbows were changed , so easiest way to do it is to go to main , pull , create a new branch from main and lower light crossbows damage by 1